app/views/shipit/stacks/_header.html.erb

```diff
@@ -36,7 +36,7 @@
     </li>
     <% if stack.deploy_url.present? %>
       <li class="nav__list__item">
-        <%= link_to 'View website', stack.deploy_url, :target => '_blank', :class => 'deploy-url' %>
+        <%= sanitize link_to 'View website', stack.deploy_url, :target => '_blank', :class => 'deploy-url' %>
```

Comment:
:+1: so can we just call `sanitize` in the argument not in the whole `link_to` method? That will make clear what is dangerous.